scripts/listChangedFiles.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+
+// Based on similar script in React
+// https://github.com/facebook/react/blob/b87aabdfe1b7461e7331abb3601d9e6bb27544bc/scripts/shared/listChangedFiles.js
+
+const execFileSync = require('child_process').execFileSync;
```

Comment:
I believe React isn't using Babel, we could have been using ES6 modules and async/await.

Reply:
Is this only for execFile or for all Sync functions in the scripts? e.g. readFile, writeFile, glob

Reply:
I would just stick with node runtime and commonJS modules. It's not worth it to always invoke babel just for ES modules IMO.